tensorrt_llm/_torch/models/modeling_mistral.py

```
@@ -309,7 +309,7 @@ def get_num_tokens_per_image(self, image_size):
             Image.new("RGB", (w, h)))
         return ncols * nrows + nrows
 
-    def __call__(self, text, images, **kwargs):
+    def __call__(self, text, images=[], **kwargs):
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)